screenpy/actions/eventually.py

```diff
@@ -121,7 +123,8 @@ def describe(self) -> str:
     def perform_as(self, the_actor: Actor) -> None:
         """Direct the actor to just keep trying."""
         if self.poll > self.timeout:
-            raise ValueError("poll must be less than or equal to timeout")
+            msg = "Poll period must be less than or equal to timeout."
+            raise UnableToAct(msg)
```

Comment:
I changed this from `ValueError` to `UnableToAct`. I standardized on this exception in this PR if an Action got stuff that it couldn't use to `perform_as` later. 

It feels weird to me that we have an `UnableToAct` exception that we were only sometimes using, it feels like a breach of an exception-abstraction barrier.

Reply:
I agree with the change.  My only concern would be doing too many things in the PR.  
Changing the exception could cause downstream users problems, which might need to be put into a higher minor version etc.

Reply:
On that note, i think there are still a couple of exceptions that aren't `ScreenPyError`-based ones raised in various places. I should probably go through and look closer.

Should i undo these changes until then?

Reply:
Already the next release is definitely going to be at least a +0.1.0 bump with all the new toys, so i think changing exceptions will fit in there.

Reply:
If that's the case, leave it in.  We'll just need to be sure to document it.